packages/liveblocks-client/src/__tests__/room.test.ts

```diff
@@ -936,11 +980,19 @@ describe("room", () => {
 
       const unsubscribe = machine.subscribe("others", (o) => (others = o));
 
+      machine.onMessage(
+        serverMessage({
+          type: ServerMsgCode.ROOM_STATE,
+          users: { 1: {} },
+        })
+      );
+
       machine.onMessage(
         serverMessage({
           type: ServerMsgCode.UPDATE_PRESENCE,
           data: { x: 2 },
           actor: 1,
+          targetActor: 0, // Setting targetActor means this is a full presence update
```

Comment:
…and `UPDATE_PRESENCE` (full) now. 2/2

Reply:
I think this litters implementation details in the tests. Perhaps this behavior should be moved into the `setupStateMachine()` or `machine.connect()` somewhere? Not super sure where this belongs. @ofoucherot tips?